code/backend/batch/AddURLEmbeddings.py

```diff
@@ -1,20 +1,21 @@
 import logging
 import traceback
 import azure.functions as func
-import sys
 
 from utilities.helpers.DocumentProcessorHelper import DocumentProcessor
 from utilities.helpers.ConfigHelper import ConfigHelper
 
-sys.path.append("..")
```

Comment:
Do we know why this was needed? Do the deployed functions still work without this?

Reply:
I don't know why this was added. (My guess would be that there was some refactoring and this was left behind). The deployed functions still work, verified both in the deployment and run locally with `poetry func start`. 

I also think it's much more robust to use an absolute path with sys.path.append as I do here in the test files